content/docs/0.9.x/reference/bridge/manage_services/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Manage Services
+description: Create, update and delete services from the Bridge
+weight: 40
+keywords: [0.9.x-bridge]
+---
+
+The Bridge provides features to make managing services more convenient. It is possible to create and delete services directly from the UI.
+The following sections explain in detail which functionalities you can use to set up your services and where to find them.
+
+This may also be interesting for you:
+
+* [Create a service with the CLI](https://keptn.sh/docs/0.9.x/reference/cli/commands/keptn_create_service/)
+
+## Create a service in Bridge
+Going to the services settings, there is a "Create service" button.
+Clicking on this button opens a view to enter a valid name for the service.
```

Comment:
... opens a form where you can enter a name for the service. A valid service name contains lowercase letters, numbers, and hyphens.
(Please re-validate if the valid service name criteria I have written is correct :) )